docs/source/science/HLS/HLSL30.ipynb

```diff
@@ -2,23 +2,87 @@
  "cells": [
```

Comment:
<p>Suggested changes, along with removing the code cell below : </p><p> </p><blockquote>To be able to access the imagery, we need to have credentials with access to the LPDAAC S3 bucket. We will create a session with such credentials multiple times in the notebook, so here we create a function for easy reuse. It wraps an AWS session in a <code>rasterio</code> AWS session.  </blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>I think we leave the code cell below, the text can be changed if you want</p>